tests/testthat/test-scales.r

```diff
@@ -370,3 +370,15 @@ test_that("scale_apply preserves class and attributes", {
   expect_false(inherits(out, "foobar"))
   expect_null(attributes(out))
 })
+
+test_that("All scale_colour_*() have their American versions", {
+  # In testthat, the package env contains non-exported functions as well so we
+  # need to parse NAMESPACE file by ourselves
+  exports <- readLines(system.file("NAMESPACE", package = "ggplot2"))
+  colour_scale_exports <- grep("scale_.*colour", exports, value = TRUE)
+  color_scale_exports <- grep("scale_.*color", exports, value = TRUE)
```

Comment:
I don't understand the regular expression. Shouldn't it be `scale_colour_.*`, or maybe `\(scale_colour_.*\)` to be safe?

Reply:
Ah, true... I'll use `export\(scale_colour_.*\)` for more precision.

``` r
exports <- readLines(system.file("NAMESPACE", package = "ggplot2"))
grep("export\\(scale_colour_.*\\)", exports, value = TRUE)
#>  [1] "export(scale_colour_binned)"     "export(scale_colour_brewer)"    
#>  [3] "export(scale_colour_continuous)" "export(scale_colour_date)"      
#>  [5] "export(scale_colour_datetime)"   "export(scale_colour_discrete)"  
#>  [7] "export(scale_colour_distiller)"  "export(scale_colour_fermenter)" 
#>  [9] "export(scale_colour_gradient)"   "export(scale_colour_gradient2)" 
#> [11] "export(scale_colour_gradientn)"  "export(scale_colour_grey)"      
#> [13] "export(scale_colour_hue)"        "export(scale_colour_identity)"  
#> [15] "export(scale_colour_manual)"     "export(scale_colour_ordinal)"   
#> [17] "export(scale_colour_steps)"      "export(scale_colour_steps2)"    
#> [19] "export(scale_colour_stepsn)"     "export(scale_colour_viridis_b)" 
#> [21] "export(scale_colour_viridis_c)"  "export(scale_colour_viridis_d)"
```

<sup>Created on 2020-07-13 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>